modules/nf-core/gmmdemux/tests/nextflow.config

```diff
@@ -0,0 +1,15 @@
+params{
+    // Optional parameters for GMM-demux - Default values
+    threshold = 0.8
+    num_cells = 35685
+    output_path = 'test'
+    examine = " example_cell_types.txt"
+    ambiguous = 0.05
+
+}
+
+process {
+    withName: GMMDEMUX {
+        ext.args = "-t ${params.threshold} --summary ${params.num_cells} -o ${params.output_path} -a ${params.ambiguous} "
```

Comment:
Nope, the -o is the parameter for the folder where the filtered matrix produced by the tool is saved. It is described as: `The path for storing the Same-Sample-Droplets (SSDs). SSDs are stored in mtx format. Requires a path argument. Default path: SSD_mtx.`
So basically, if we change the path, it's a bit more difficult to capture it with NextFlow, then by giving it just a name, the tool creates the folder, which is captured by Nextflow and added to our outputs. Should I just leave it as Val in the inputs? or what would you suggest about this one?

Reply:
Okay, so with `-o` you can determine the name of the folder the tool generates? I'm also not entirely sure what the best way to deal with this is. 
My intuition right now would be to use the flag in the main script to publish the corresponding files to a "$prefix" subdirectory. That of course would mean that a user could not use the flag anymore, but I guess that is the entire point of the prefix variable?
It might be worth looking at some other modules that have renaming flags and how they deal with it or asking in the nf-core slack what the recommended option is.

Reply:
It is typical to fix the output path to be `$prefix` in the module logic, I think that is appropriate here.

Reply:
I managed to have all outputs in the same folder by using the `-o .` so we don't end up with 2 different folders of results, is that ok?